src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1325,22 +1325,34 @@ private AdsDataProvider() {
     };
   }
 
-  @DataProvider
-  public static Object[][] adsVUAPTopDesktop() {
-    return new Object[][]{
-      {
-        AdsContent.TOP_LB,
-        new Page("project43", "SyntheticTests/VUAP")
-      }
-    };
-  }
+    @DataProvider
+    public static Object[][] adsVUAPTopDesktop() {
+        return new Object[][]{
+                {
+                        new Page("project43", "SyntheticTests/VUAP"),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SyntheticTests/VUAP" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)